### ISSUE TITLE
How to create label text file for image without any object?

### ISSUE THREAD

**Issue by witignite** [tzutalin/labelImg#659]
I notice that the "Save" button is grayed out when no bounding box is selected on the image. How to create a label text file (with empty content) for images without any objects? I'm trying to look into the code but kind of getting lost. Could anyone point me to the code where I should modify to "ungray" the Save button?

**Comment by 5588-jmc** [comment]
You can try to skip this one and rearrange after the file is generated.

**Comment by witignite** [comment]
@5588-jmc As you said, we could skip an image that doesn't contain objects. Once we finished labeling, we would have a folder containing label text files for each image that we labelled, and we could generate empty label text files if the label text file is not present in that folder. However, this assumes that we label all the images in a folder that contains objects. But in practice, I want to skip some images because they look just the same, or they provide no new information. So I think being able to explicitly save the label file would be helpful.

**Comment by kinoute** [comment]
@witignite Did you find a solution? I'm looking for someone similar. Since the absence of an annotation file could mean two things: either there is nothing to detect on the image, either the image wasn't labelled yet, it's hard to track what we analysed already and what we didn't.

**Comment by sevgiserra** [comment]
@kinoute When the image is opened in labelImg, you can create an unnecessary bounding box and select a random class and then delete the bounding box. This way the save button is usable and it saves an empty txt file for that image.

**Comment by kinoute** [comment]
@sevgiserra Unfortunately, this is not really a scalable solution (I have to work on 10k+ images). For the moment, I forked labelImg to force the creation/save of the annotation file no matter what, it seems to work. 

I'm going to try to plug a pre-trained model I made on a few samples of my dateset to generate the bounding boxes coordinates, that way I will just have to verify the predictions instead of annotate.
